build_runner/lib/src/build_script_generate/build_script_generate.dart

```diff
@@ -47,7 +47,8 @@ Future<String> _generateBuildScript() async {
       ..writeln('// ignore_for_file: directives_ordering')
       ..writeln(library.accept(emitter));
 
-    return DartFormatter().format(content.toString());
+    return DartFormatter(languageVersion: DartFormatter.latestLanguageVersion)
```

Comment:
Really this should be the language version of the containing package.

Or, we could put a language version comment in this file and then hard code that version?

Reply:
See my latest commit. We'd have to create a `Library` element or something for the current package using the analyzer API. I wish there was a cleaner way to do this...

Reply:
We probably should be adding a language version comment anyways, we are generating code that we only know will work as of some language version anyways.

Reply:
done